src/pipeline/src/etl/error.rs

```diff
@@ -582,6 +582,12 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+    #[snafu(display("Field is required for dispatcher"))]
+    FieldRequiredForDispatcher,
+    #[snafu(display("table_part is required for dispatcher rule"))]
+    TablePartRequiredForDispatcherRule,
+    #[snafu(display("value is required for dispatcher rule"))]
+    ValueRequiredForDispatcherRule,
```

Comment:
sugg: add `Location` to them

Reply:
I removed it because this validation error seems obvious and only used once